### PR TITLE
Add basic Node JS plugin support

### PR DESCRIPTION
**Pull request by mattcompiles**
## Motivation

This PR adds basic Node JS plugin support to Atlaspack V3. I've stripped back the code to not bother handle dependencies at all. This requires a bunch more changes which I have done in [another branch](https://github.com/atlassian-labs/atlaspack/tree/node-plugin-html) but I'm not sure if we really want to continue supporting dependencies in JS plugins going forward. If we decide we'd like to add this feature we can follow up with that. 

For now this PR is enough to get a basic transformer like `@atlaspack/transformer-svg` working which I have enabled an integration test for in this PR. There's still a bunch more things to add like invalidations and source-maps but this is a good start. 

## Checklist

- [] Existing or new tests cover this change

### CODE REVIEW

File: packages/core/core/src/atlaspack-v3/worker/worker.js [hunk 1]
```diff
@@ -1,26 +1,87 @@
...
+
+    try {
+      if (transformer.parse) {
+        // $FlowFixMe
+        let ast = await transformer.parse({asset: assetCompat});
+        // $FlowFixMe
+        assetCompat.setAST(ast);
+      }
+
+      // $FlowFixMe
```
Comment (yamadapc) [inline packages/core/core/src/atlaspack-v3/worker/worker.js:45]: What's wrong?

**Review by yamadapc — APPROVED** [review]

File: crates/atlaspack_plugin_rpc/src/plugin/transformer.rs [hunk 2]
```diff
@@ -9,40 +10,81 @@ use atlaspack_core::plugin::TransformResult;
...
-    Ok(TransformResult {
+    let asset_env = asset.env.clone();
+    let stats = asset.stats.clone();
+    let run_transformer_opts = RpcTransformerOpts {
+      // TODO: Make this not bad
+      resolve_from: self.plugin.resolve_from.to_str().unwrap().to_string(),
+      specifier: self.plugin.package_name.clone(),
+      // TODO: Pass this just once to each worker
+      options: self.plugin_options.clone(),
       asset,
```
Comment (yamadapc) [inline crates/atlaspack_plugin_rpc/src/plugin/transformer.rs:57]: The nicest thing would be to make this be a non serialized object. 

So we would create an object from Rust with getter/setter fields that modify or access that directly on demand.

Comment (mattcompiles) [reply crates/atlaspack_plugin_rpc/src/plugin/transformer.rs:57]: Interesting idea. I'll look into that.

File: packages/core/core/src/atlaspack-v3/worker/worker.js (hunk 1 above)
Comment (mattcompiles) [reply packages/core/core/src/atlaspack-v3/worker/worker.js:45]: Transformers currently expect a larger API which I'm not yet providing.

**Pull request merged**